wReleaseNotes/2024-06_ReleaseNotes.md

```diff
@@ -3,19 +3,48 @@ title: Release Notes
 filename: "2024-06_ReleaseNotes.md"
 layout: custom
 keywords: [change log, updates, versions, history, what's new]
-headings: ["June 2024", "Documentation Updates"]
-links: []
+headings: ["June 2024", "Excel Addin v2.5.1.1", "Portal Site", "Documentation Updates"]
+links: ["/wAbout/SingleUser.html", "/wAbout/SharedComputer.html", "/wPortal/Roles.html", "/wPortal/FolderPermissions.html"]
 image_dir: ""
 images: []
 description: Release notes for this month's updates
 ---
 * * *
 
-## May 2024
+## June 2024
+
+### Excel Addin v2.5.1.1
+
+_Released 6/4/24_
+
+- 🐞 Wrapped decimal conversion in try/catch where excel versions were being read in addin manager. This fixes a bug where setting the region on your machine causes the addin manager to not open.
```

Comment:
capitalize Excel

Reply:
probably Addin Manager also

Reply:
there's also something weird about "causes the addin manager not to open". would be better/clearer to say "prevents the addin manager from opening"